src/components/templates/Audience/Audience.tsx

```diff
@@ -46,7 +46,7 @@ interface ChatOutDataType {
 const SEAT_SIZE = "var(--seat-size)";
 const SEAT_SIZE_MIN = "var(--seat-size-min)";
 
-const VIDEO_MIN_WIDTH_IN_SEATS = 8;
+const VIDEO_MIN_WIDTH_IN_SEATS = 10;
```

Comment:
Why did we decide to make this `10`? I didn't see any discussion/etc related to it on the original issue, nor explanation here explaining it.

Have we considered what the flow on effects of this change are going to be, and whether we will need to change any of our other defaults/min/max values/etc? (including in admin v1/v2, validations, etc)

Reply:
Before this change:
![image](https://user-images.githubusercontent.com/753891/111232210-c0763a00-863e-11eb-912d-42b0caa57c41.png)

After this change:
![image](https://user-images.githubusercontent.com/753891/111232233-cd932900-863e-11eb-978a-0a8bf7638359.png)

Reply:
The video being tiny was mentioned by @jess-sparkles in the original issue
 `The video is also super tiny and hard to see/ watch. Is there a way to make iFrame in the center larger and the seats fill the screen more?`

Reply:
The iframe was tiny because the whole ‘seats block’ was only using a small chunk of the screen. So the changes to make it bigger/use the whole of the screen (by increasing the seat size) already make the iframe bigger.

IMO the outer rows of the seats look too ‘squished’ now, it felt way more balanced as was.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1267/commits/3ce92bcfe8cf07d9ad37c72f1fde352e53555bd9